samples/quickstart.js

```diff
@@ -27,29 +27,36 @@ const client = new textToSpeech.TextToSpeechClient();
 // The text to synthesize
 const text = 'Hello, world!';
 
-// Construct the request
-const request = {
-  input: {text: text},
-  // Select the language and SSML Voice Gender (optional)
-  voice: {languageCode: 'en-US', ssmlGender: 'NEUTRAL'},
-  // Select the type of audio encoding
-  audioConfig: {audioEncoding: 'MP3'},
-};
-
-// Performs the Text-to-Speech request
-client.synthesizeSpeech(request, (err, response) => {
-  if (err) {
-    console.error('ERROR:', err);
-    return;
-  }
-
-  // Write the binary audio content to a local file
-  fs.writeFile('output.mp3', response.audioContent, 'binary', err => {
-    if (err) {
-      console.error('ERROR:', err);
-      return;
-    }
-    console.log('Audio content written to file: output.mp3');
+function synthesizeSpeech(request = {}) {
+  return new Promise((resolve, reject) => {
+    // Performs the Text-to-Speech request
+    client.synthesizeSpeech(request, (err, response) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(response);
+      }
+    });
   });
+}
+
+async function main() {
+  // Construct the request
+  const request = {
+    input: {text: text},
+    // Select the language and SSML Voice Gender (optional)
+    voice: {languageCode: 'en-US', ssmlGender: 'NEUTRAL'},
+    // Select the type of audio encoding
+    audioConfig: {audioEncoding: 'MP3'},
+  };
+
+  const response = await synthesizeSpeech(request);
+  // Write the binary audio content to a local file
+  fs.writeFileSync('output.mp3', response.audioContent, 'binary');
+  console.log('Audio content written to file: output.mp3');
+}
+
+main().catch(err => {
```

Comment:
shorten to `main().catch(console.error);`

Reply:
implemented requested changes.